docs/quickstart/install.md

```diff
@@ -258,6 +258,90 @@ above by running the following before any other CLI commands:
 - `tanzu config eula accept` to accept the EULA.
 - To set the CEIP participation status for automation, the environment variable `TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER` can be set to `No` or `Yes`.
 
+### Essentials plugin group
+
+Introducing a plugin group named vmware-cli/essentials, which includes all the necessary prerequisite plugins for tanzu cli experience.
+The initial offering of vmware-cli/essentials will include the “telemetry” plugin but the list could grow in the future.
+
+Essential plugins will be installed on the system to enhance the overall tanzu CLI experience.
+For instance, the Telemetry plugin should be consistently installed to enable the collection and transmission of telemetry data.
+
+Essentials plugins installed when user accepts EULA
+
+```shell
+> tanzu plugin list
+
+? You must agree to the VMware General Terms in order to download, install, or
+use software from this registry via Tanzu CLI. Acceptance of the VMware General
+Terms covers all software installed via the Tanzu CLI during any Session.
+“Session” means the period from acceptance until any of the following occurs:
+(1) a change to VMware General Terms, (2) a new major release of the Tanzu CLI
+is installed, (3) software is accessed in a separate software distribution
+registry, or (4) re-acceptance of the General Terms is prompted by VMware.
+
+To view the VMware General Terms, please see https://www.vmware.com/vmware-general-terms.html
+
+If you agree, the essentials plugins will be installed that is necessary for tanzu cli experience
+
+Do you agree to the VMware General Terms?
+ > Yes
+
+[i] The tanzu cli essential plugins have not been installed and are being installed now. The install may take a few seconds.
+
+Standalone Plugins
+  NAME       DESCRIPTION              TARGET  VERSION  STATUS
+  telemetry  telemetry functionality  global  v0.0.9   installed
+```
+
+With each update of this essential plugin (or any future one added to the group), a new version of this plugin group will be released. The Tanzu CLI Core team will oversee its maintenance.
```

Comment:
We do not need to mention the last statement in the public installation document.

```suggestion
With each update of this essential plugin (or any future one added to the group), a new version of this plugin group will be released.
```